scripts/stop.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 
-ps aux | grep ksql-server-start | grep -v "grep ksql" | awk '{print $2}' | xargs kill -15
+pkill ksql-server-start
```

Comment:
This line might be a remnant from a previous version of cp-demo.  Please double-check if this line can be completely removed.

Reply:
I don't know the context of why it was added originally, have a jira or some details?

Reply:
@rspurgeon you can go back to the earlier branches and see how it may have been used.

Reply:
Per our DM and a local test, I removed the unecessary pkill command from stop.sh